pandas/core/generic.py

```diff
@@ -4003,18 +4004,62 @@ def asfreq(self, freq, method=None, how=None, normalize=False):
             For PeriodIndex only, see PeriodIndex.asfreq
         normalize : bool, default False
             Whether to reset output index to midnight
+        fill_value: scalar, optional
+            value to use for missing values, applied during upsampling
```

Comment:
Can you add a similar sentence as for `method`: "this does not fill NaNs that were already present"?
(can you also capitalize the first word of the sentence?